src/Uno.UWP/ApplicationModel/Contacts/ContactQueryOptions.Android.cs

```diff
@@ -0,0 +1,38 @@
+#if __ANDROID__
```

Comment:
There is no need to set this android specific, the behavior is cross-platform.

Reply:
Ok.